hack/doc/role-permissions2md.py

```diff
@@ -0,0 +1,153 @@
+    "dynatrace-operator": "Dynatrace Operator",
+    "dynatrace-kubernetes-monitoring": "Dynatrace Kubernetes Monitoring (ActiveGate)",
+    "dynatrace-webhook": "Dynatrace webhook server",
+    "dynatrace-oneagent-csi-driver": "Dynatrace CSI driver",
+    "dynatrace-activegate": "Dynatrace Kubernetes Monitoring (ActiveGate)",
+    "dynatrace-dynakube-oneagent": "Dynatrace OneAgent"
+}
+
+def get_apis(rule):
+    apis = rule.get('verbs')
+
+    api_string = ""
+    for api in apis:
+        if (len(api_string) > 0):
+            api_string += "/"
+        api_string += apiTerms[api]
+
+    return api_string
+
+def multiline_codestyle_block(stringList):
+    result_string = ""
+    for entry in stringList:
+        if (len(result_string) > 0):
+            result_string += "<br />"
+        if len(entry) > 0:
+            result_string += f"`{entry}`"
+        else:
+            result_string += f"`-`"
+    return result_string
+
+def get_resource_names(rule):
+    resource_names = rule.get('resourceNames')
+    if resource_names == None:
+        return ""
+    return multiline_codestyle_block(resource_names)
+
+def get_api_groups(rule):
+    api_groups = rule.get("apiGroups")
+    if api_groups == None:
+        return ""
+    return multiline_codestyle_block(api_groups)
+
+def create_role_table(role):
+    print('|Resources accessed |API group |APIs used |Resource names |')
+    print('|------------------ |--------- |--------- |-------------- |')
+
+    for rule in role['rules']:
+        resources = rule.get('resources')
+        if resources != None:
+            for resource in resources:
+                apis = get_apis(rule)
+                resource_names = get_resource_names(rule)
+                api_gropus = get_api_groups(rule)
+                print(f"|`{resourceTerms[resource]}` |{api_gropus} |{apis} |{resource_names} |")
+
+def convert_cluster_roles_to_markdown(role):
+    print(f"\n## {sectionTitles[role['metadata']['name']]} (cluster-wide)\n")
+    create_role_table(role)
+
+def convert_roles_to_markdown(role):
+    print(f"\n## {sectionTitles[role['metadata']['name']]} (namespace {role['metadata']['namespace']})\n")
+    create_role_table(role)
+
+def main():
+    parser = argparse.ArgumentParser(description="Convert ClusterRoles and Roles to MD permission table",
+                                    formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument("src", help="Source K8S manifest")
+    args = parser.parse_args()
+
+    parsed_url = urlparse(args.src)
+    if bool(parsed_url.scheme):
+        file = urlopen(args.src)
+    else:
+        file = open(args.src, "r")
+
+    docs = yaml.safe_load_all(file)
+
+    # for manifest in docs:
+    #     print(f"{manifest['metadata']['name']} {manifest['kind']}")
+    manifests = []
+
+    for manifest in docs:
+        manifests.append(manifest)
+
+    for manifest in manifests:
+#        print(f"\n{manifest['metadata']['name']} {manifest['kind']}")
```

Comment:
let's remove commented code. Or you can add option to args `--debug` default = False, and print some debug lines if you wish.